model/AssessmentResultsService.php

```diff
@@ -187,6 +189,20 @@ public function getPrintableRubric(DeliveryExecutionInterface $deliveryExecution
         return $rubrics;
     }
 
+    /**
+     * Get the configurable url parts or default url to proctoring
+     */
+    public function getScoreReportUrlParts()
+    {
+        return $this->hasOption(self::OPTION_SCORE_URL) && is_array($this->getOption(self::OPTION_SCORE_URL))
+            ? $this->getOption(self::OPTION_SCORE_URL)
+            : array(
+                'extension' => 'taoProctoring',
```

Comment:
I would prefer having this kind of config setup by default in in taoProctoring/config/default/AssessmentResults.conf.php and overwrite by nccer-delivery, it would make things easier to detect the default config.